app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/BaseActivity.java

```diff
@@ -81,6 +90,49 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
         return super.onKeyUp(keyCode, event);
     }
 
+    @Override
+    public void navigateToSessionDetail(@NonNull Session session) {
+        raiseNavigationError();
+    }
+
+    @Override
+    public void navigateToFeedbackPage(@NonNull Session session) {
+        raiseNavigationError();
+    }
+
+    @Override
+    public void navigateToSponsorsPage() {
+        raiseNavigationError();
+    }
+
+    @Override
+    public void navigateToContributorsPage() {
+        raiseNavigationError();
+    }
+
+    @Override
+    public void navigateToLicensePage() {
+        raiseNavigationError();
+    }
+
```

Comment:
All activity transition method throws an exception in `BaseActivity`.
So we can know the fact that we accidentally dispatch request which navigates to unreachable view by this exception.